### DISCUSSION TITLE
Need help with the plug in

### DISCUSSION THREAD

**Discussion by Normie51** [JustOptimize/return-ShowHiddenChannels#225] [Q&A] [answered]
I can see voice channels, but I can't see text channels 
In the preview it shows you can see both but I can only see the vcs
![image_2024-11-13_204236208](https://github.com/user-attachments/assets/3c8bba21-7c77-410e-aba2-75382ac2cfb2)

**Comment by JustOptimize** [comment, marked as answer]
Right click on the server and click show all channels
